### DISCUSSION TITLE
Huge Breakthrough

### DISCUSSION THREAD

**Discussion by AnOrangeCone** [3kh0/ext-remover#1032] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
succesfully unenrolled and got into dev mode on v124 (using sh1mmer legacy) my board is octopus phaser360 and I'm using a lenovo 500e second gen

**Comment by AnOrangeCone** [comment]
tutorial:
be on v118 or bellow (this might work on v119+ i dont know since I didnt try)
install sh1mmer on the recovery drive and boot into it with the method on sh1mmers website
unenroll and enable dev mode (WAIT THE 5 MINUTES) (i forgot if i unenrolled with cryptosmite or sh1mmer inside of sh1mmer I believe it was cryptosmite if that doesnt work use the sh1mmer one)
then hope and pray it works
follow the rest of the sh1mmer steps and boot into chromeos
next step is to update your chromebook to 124 and there it is
btw I didnt do any hardware mods this was entirely software
my kernel version is 2 that might be important since it may not work on v3 and im on stable channel octopus with phaser360
this might now work for you but it did for me if it doesnt work for you downgrade to v118 and if it still doesnt work it 100% works on v114 then you can update back and use fakemurk if you want (I dont recommend using fakemurk though use sh1mmers method to re enroll when needed and just re un enroll when done its much safer that way)
YOU COULD TRY TO INSTALL WINDOWS BUT I CANT TEST IT BECAUSE MY CHROMEBOOK IS BAD)
this isnt a good tutorial but its the basics

**Reply by AnOrangeCone** [reply]
i know this isnt technically unenrolling on v124 but its the best we have for now until a real method is found

**Reply by CGRmags** [reply]
isnt this just cyptosmite?

**Reply by AnOrangeCone** [reply]
kinda but I used sh1mmer

**Comment by nathblade16** [comment]
you should probably blur your email

**Reply by AnOrangeCone** [reply]
ima just delete this post
